packages/@react-aria/focus/test/FocusScope.test.js

```diff
@@ -931,6 +967,42 @@ describe('FocusScope', function () {
       // and wrap is false
       expect(document.activeElement).toBe(item1);
     });
+
+    it('should move focus backward and allow users to skip certain elements', function () {
+      function Test() {
+        return (
+          <FocusScope>
+            <Item data-testid="item1" />
+            <Item data-testid="item2" data-skip />
+            <Item data-testid="item3" />
+          </FocusScope>
+        );
+      }
+
+      function Item(props) {
+        let focusManager = useFocusManager();
+        let onClick = () => {
+          focusManager.focusNext({
```

Comment:
should this be focusPrevious()?

I'm wondering if there should be four items, so we guarantee a skip because right now these look and work similar.